examples/sample_measurement/pyproject.toml

```diff
@@ -6,15 +6,21 @@ authors = ["National Instruments"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-ni-measurementlink-service = {version = "^1.1.0-dev2", allow-prereleases = true}
+ni-measurementlink-service = {version = "^1.2.0-dev0", allow-prereleases = true}
 click = ">=7.1.2, !=8.1.4" # mypy fails with click 8.1.4: https://github.com/pallets/click/issues/2558
 
 [tool.poetry.group.dev.dependencies]
 mypy = ">=1.0"
 grpc-stubs = "^1.53"
+types-protobuf = "^4.21"
 # Uncomment to use prerelease dependencies.
-# ni-measurementlink-service = {path = "../..", develop = true}
+ni-measurementlink-service = {path = "../..", develop = true}
 
 [build-system]
 requires = ["poetry-core>=1.2.0"]
 build-backend = "poetry.core.masonry.api"
+
+[[tool.mypy.overrides]]
+module = [
+    "ni-measurementlink-service.*",
+]
```

Comment:
This has no configuration keys so delete it. The module key identifies which module the other keys apply to.
https://mypy.readthedocs.io/en/stable/config_file.html#using-a-pyproject-toml-file

Reply:
Removed as suggested.